### ISSUE TITLE
Policy on intake for larger data assets

### ISSUE THREAD

**Issue by jlstevens** [pyviz-topics/examples#14]
The [opensky](https://github.com/pyviz-topics/examples/blob/master/opensky.ipynb) notebook is an example of a topic that requires a large data file (`opensky.parq`). I would like to suggest using `intake` in such cases instead of the current approach that will need updating anyway (right now it relies on datashader's system for getting data files).

**Comment by jbednar** [comment]
I think using intake is a good default, but we should probably have a couple of examples that show other ways of getting data files.

**Comment by jsignell** [comment]
I have an approach that I've already been doing.

**Comment by jbednar** [comment]
We should make some of the examples use Intake for its own sake, but I think the anaconda-project.yml handles the typical case.

**Comment by jsignell** [comment]
Just to circle back. I tried to use intake for the 1-billion osm data point case in #20, and ended up bailing because the download often takes longer than the 10 minutes before a notebook cell times out. I kind of decided that for really big downloads ( ~>3GB) we probably want to just tell the user to download the file rather than handling it behind the scenes. I have set up the infrastructure to use intake on these things generally (#22). The benefit to using intake over anaconda-project download is that the download only happens at the moment it is needed, this ends up being sort of annoying in AE because the deployment doesn't download the data on deployment.
